public/components/layout.js

```diff
@@ -65,3 +65,169 @@ class GeoFooter extends HTMLElement {
 }
 
 customElements.define("geo-footer",GeoFooter)
+
+class UserResource extends HTMLElement {
+    #uriInputTmpl = `
+        <div id="supplyURI" class="card">
+            <header>
+                Supply an existing Web Resource URI.
+            </header>
+            <div>
+                <label>Object URI</label><input id="objURI" type="text" />
+            </div>
+            <footer>
+                <input id="uriBtn" type="button" class="button primary" value="Use This URI" />
+            </footer>
+        </div>
+
+        <div id="confirmURI" class="card is-hidden notfirst">
+            <header>Resolved URI</header>
+            <div>
+                <input id="confirmUriBtn" type="button" class="button primary" value="Confirm URI" />
+                <div id="uriPreview">
+
+                </div>
+            </div>
+            <footer>
+
+            </footer>
+        </div>`
+
+    connectedCallback() {
+        this.innerHTML = this.#uriInputTmpl
+        localStorage.removeItem("providedURI")
```

Comment:
It seems a little weird to save this to localStorage. Maybe a history of previous entries that resolved... 
This strays towards playground when I think we have just a need of using the history of the application.